### PR TITLE
add docblock for method

### PR DESCRIPTION
**Pull request by michaelKaefer**
| Q             | A
| ------------- | ---
| Branch?       | master for features
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no 
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Comment by xabbuh** [comment]
I am :-1: here. The added docblock does not add any additional value compared to the method name.

**Comment by theofidry** [comment]
@xabbuh here it does provide an info, that the `$url` is a string. I'm not for the comment in the phpdoc

**Comment by xabbuh** [comment]
Oh indeed, we should keep the param type, but remove the description.

**Comment by michaelKaefer** [comment]
@xabbuh @theofidry Thanks, I made another commit. :+1:

File: src/Symfony/Component/HttpKernel/Profiler/Profile.php [hunk 1]
```diff
@@ -69,16 +65,14 @@ public function getToken()
     }
 
     /**
-     * Sets the parent token.
+     * @param self
```
Comment (localheinz) [inline src/Symfony/Component/HttpKernel/Profiler/Profile.php]: How about adding the parameter name here, too?

**Review by localheinz — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/HttpKernel/Profiler/Profile.php (hunk 1 above)
Comment (xabbuh) [reply src/Symfony/Component/HttpKernel/Profiler/Profile.php]: This docblock is actually not need as the parameter has a type hint.

**Comment by fabpot** [comment]
Thanks @michaelKaefer for this pull request. I'm going to close it as we try to minimize the changes that do not add value (to ease merging older branches into newer ones). Here, I don't think the changes are worth as the information is already part of the PHP signature. I agree that removing some descriptions might be useful (and we would probably not add them if they were not present today), but they don't hurt either.

**Pull request closed without merging**